driver/src/price_finding/naive_solver.rs

```diff
@@ -65,7 +65,7 @@ impl PriceFinding for NaiveSolver {
         state: &State
     ) -> Result<Solution, PriceFindingError> {
         // Initialize trivial solution
-        let mut prices: Vec<u128> = vec![1; 1 + TOKENS as usize];
+        let mut prices: Vec<u128> = vec![1; TOKENS as usize];
```

Comment:
@fleupold - Not sure how to get the commit has from this, but you could cherry-pick this if you haven't already.